docs/sections/pipeline_samples/tutorials/generate_textcat_dataset.ipynb

```diff
@@ -0,0 +1,1005 @@
+{
```

Comment:
<p>should we provide an option where people simply don't use argilla?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/distilabel/pull/949/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>